mobile/src/bundles/common/enums/navigation/onboarding-screen-name.enum.ts

```diff
@@ -1,6 +1,9 @@
 const OnboardingScreenName = {
     PROFILE: 'Profile',
     BSA_BADGES: 'BSA Badges',
+    SKILLS_PROJECTS: 'Skills and projects',
+    CV_CONTACTS: 'CV and contacts',
+    PREVIEW: 'Preview',
```

Comment:
We will have some merging conflicts because I've also created these screens.

Reply:
I only added the names for the automatic calculation of the number of steps in STEP 01/05. I think this conflict will be easy to resolve later.

Reply:
Yeah